shell/pom.xml

```diff
@@ -51,14 +51,18 @@
       <groupId>commons-io</groupId>
       <artifactId>commons-io</artifactId>
     </dependency>
-   <dependency>
-     <groupId>org.apache.httpcomponents</groupId>
-     <artifactId>httpclient</artifactId>
-   </dependency>
-   <dependency>
+    <dependency>
+      <groupId>org.apache.httpcomponents</groupId>
+      <artifactId>httpclient</artifactId>
+    </dependency>
+    <dependency>
```

Comment:
please fix the spacing on this dependency

Reply:
I added the spaces just because these lines were incorrectly aligned. Please compare with the lines above and below.